Tools/SandboxTest.ps1

```diff
@@ -0,0 +1,298 @@
+# Parse arguments
+
+Param(
+  [Parameter(Position = 0, HelpMessage = "The Manifest to install in the Sandbox.")]
+  [String] $Manifest,
+  [Parameter(Position = 1, HelpMessage = "The script to run in the Sandbox.")]
+  [ScriptBlock] $Script,
+  [Parameter(HelpMessage = "The folder to map in the Sandbox.")]
+  [String] $MapFolder = $pwd,
+  [switch] $SkipManifestValidation
+)
+
+$ErrorActionPreference = "Stop"
+
+$mapFolder = (Resolve-Path -Path $MapFolder).Path
+
+if (-Not (Test-Path -Path $mapFolder -PathType Container)) {
+  Write-Error -Category InvalidArgument -Message 'The provided MapFolder is not a folder.'
+}
+
+# Validate manifest file
+
+if (-Not $SkipManifestValidation -And -Not [String]::IsNullOrWhiteSpace($Manifest)) {

```

Comment:
Thanks for the suggestion, but I kind of like the PowerShell verbosity. To me, it would not make sense use `!` instead of `-Not` while keeping `-And`.

Reply:
Thanks for the suggestion, but I kind of like the PowerShell verbosity. To me, it would not make sense use `!` instead of `-Not` while keeping `-And`.